test/integration/buyers/accounts_controller_test.rb

```diff
@@ -91,6 +91,13 @@ def test_show
       assert_match 'Alaska Application App', response.body
     end
 
+    test "memeber user without billing permission cannot view invoice" do
+      @provider.settings.allow_finance!
+      buyer = FactoryBot.create(:buyer_account, provider_account: provider)
+      get admin_buyers_account_invoices_path(buyer)
+      assert_response :forbidden
+    end
```

Comment:
Good, but this should test also:
1. When the member user has finance permission, it should work!
2. You shouldn't test only 'read/show' but also the rest of actions